examples/fullstack-simple-nextjs/app/api/db.ts

```diff
@@ -81,6 +81,9 @@ function writeDatabaseFile(filePath: string, db: Database) {
   return writeFile(filePath, data);
 }
 
+const SERVER_SETUP =
+  "Ki5T3U21n_UFhcGP5fR_mUstYEItKRsjEJ2UnvvrGasT3wvjdtbluS0BsiR_LtzqX8YLrCyWqQOJDf7cL0C8DRr9pGFBSx0X22ZujePyWpdq4VddBSPm0vIXepDJEBYJjRxmRNh09pLGxk_Y9bQuFWjYInqjcy_zOVsMHPHUkQw";
```

Comment:
why a newline here?

Reply:
This is from prettier, it's formatting it like that because of the length.